src/main/java/kg/jarkyn/cobspecserver/ListenerSocket.java

```diff
@@ -0,0 +1,31 @@
+package kg.jarkyn.cobspecserver;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class ListenerSocket implements Listener {
+    private ServerSocket serverSocket;
+    private boolean accepting = true;
```

Comment:
No need for a setter.
It's just convention, as CJ mentioned, to initialise an object in the constructor. Static initialisers can get smelly.